src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/template.jelly

```diff
@@ -84,6 +84,10 @@
             <f:checkbox/>
         </f:entry>
 
+        <f:entry title="${%Run slave in exclusive mode}" field="exclusiveMode">
+            <f:checkbox/>
+        </f:entry>
```

Comment:
Help page is here src/main/webapp/help/system-config/master-slave/usage.html
jelly: hudson/slaves/DumbSlave/configure-entries.jelly
existed form: lib/form/slave-mode.jelly
Just reuse it and change variable to real Mode object.